src/icd11mms-exact-matches.sparql

```diff
@@ -0,0 +1,22 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix ECO: <http://purl.obolibrary.org/obo/ECO_>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+prefix sssom: <https://w3id.org/sssom/>
+
+SELECT ?cls ?xref
+WHERE {
+  VALUES ?mapping_pred { oboInOwl:hasDbXref }
+
+  ?cls a owl:Class;
+    ?mapping_pred ?xref .
+
+  ?xref_anno a owl:Axiom ;
+    owl:annotatedSource ?cls ;
+    owl:annotatedProperty ?mapping_pred ;
+    owl:annotatedTarget ?xref ;
+    ECO:0000218 ?mapping_precision_string .
+
+  FILTER(STRSTARTS(STR(?mapping_precision_string), "- E (Exact mapping: the two concepts are equivalent)."))
+  FILTER(STRSTARTS(STR(?xref), "ICD-11:"))
```

Comment:
## FYI: `icd11mms-exact-matches.sparql`
I tried using OAK at first, but from what I can see, `SqlImplementation` does not support querying axioms for 'mapping triples' (e.g. `oboInOwl:hasDbXref`), only 'relationship triples'  (e.g. `rdfs:subClassOf`).

Reply:
This actually ended up not being necessary. I did not need to use it to create the SSSOM output. I only needed the nomenclature file, not the OWL. But I also used this in an analysis to compare mappings between those two sources.

Reply:
I think ample time has been given to look at this, and a determination was made that this file is not necessary, so gonna mark this resolved.